packages/babel-plugin-open-source/babel.js

```diff
@@ -33,26 +33,46 @@ module.exports = declare(api => {
       if (process.env.NODE_ENV !== 'development') return
 
       const location = path.container.openingElement.loc
-      let editorURLProtocol = "vscode"
+      let url = null;
+      let editor = state.opts?.editor?.toLowerCase() || 'vscode';
 
       // the element was generated and doesn't have location information
       if (!location) return
 
       try {
         if(state.opts.envPath) {
-          editorURLProtocol = dotenv.parse(fs.readFileSync(filePath.resolve(process.cwd(),state.opts.envPath), 'utf8'))?.BABEL_OPEN_SOURCE_EDITOR;
+          editor = dotenv.parse(fs.readFileSync(filePath.resolve(process.cwd(),state.opts.envPath), 'utf8'))?.BABEL_OPEN_SOURCE_EDITOR;
         }
-      }  catch (error) {}
+      }  catch (error) {
+        console.error("ERROR in Babel Plugin Open Source", error)
```

Comment:
Would be nice to know what went wrong 😅 And how to recover from it

Reply:
The error would from path.resolve saying .env not found..

On error, it would anyways pickup vscode as default editor.

But I can surely right a helpful message with the error.